bundles/org.eclipse.ui.ide.application/src/org/eclipse/ui/internal/ide/application/IDEApplication.java

```diff
@@ -727,6 +729,17 @@ protected ReturnCode checkValidWorkspace(Shell shell, URL url) {
 			MessageDialogWithToggle dialog = new MessageDialogWithToggle(shell, title, null, message, severity,
 					buttonLabelToId, 0, IDEWorkbenchMessages.IDEApplication_version_doNotWarnAgain, false) {
 				@Override
+				protected Control createDialogArea(Composite parent) {
+					Composite composite = (Composite) super.createDialogArea(parent);
+					Button toggle = getToggleButton();
+					if (toggle != null && !toggle.isDisposed()) {
+						GridData toggleData = new GridData(SWT.BEGINNING, SWT.TOP, false, false);
+						toggleData.horizontalIndent = 50;
```

Comment:
can you explain this "magic number"?

Reply:
May this 50 be wrong on other OSes?

Reply:
I checked on macOS. Looks like this:
<img width="670" height="188" alt="image" src="https://github.com/user-attachments/assets/2a6c70e2-ebdb-4545-aef6-d7eb4eb1f0c0" />

Anyway I am still not happy with this magic number. Can this be improved?